package.json

```diff
@@ -100,6 +100,13 @@
           },
           "markdownDescription": "Optional list of properties to pass along to the Metals server. By default, the environment variable `JAVA_OPTS` and `.jvmopts` file are respected. Each property needs to be a separate item.\n\nExample: `-Dhttps.proxyHost=…`, `-Dhttps.proxyPort=…` or `-Dmetals.statistics=all`"
         },
+        "metals.ammoniteJvmProperties": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "markdownDescription": "Optional list of JVM properties to pass along to the Ammonite server. By default, the environment variable `JAVA_OPTS` and `.jvmopts` file are respected. Each property needs to be a separate item.\n\nExample: `-Xmx4G` or `-Xms1G`"
```

Comment:
also a 'verify' comment, if you decide to ignore JAVA_OPTS/.jvmopts it needs to be updated here to, if not then all good :)

Reply:
Also updated